base/osutils.jl

```diff
@@ -1,20 +1,26 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
-"""
-    OS_NAME
+let UNAME = ccall(:jl_get_UNAME, Any, ())
+    # change some `uname -s` values to "friendly" names
+    # for internal usage
```

Comment:
Only fatal if you would like win32 to cease to exist. Clearly Microsoft does not feel that way, and WSL will be more useful than midipix for some time to come. midipix is also GPL licensed at the moment which I consider to be a similarly fatal flaw for people who would like to create Windows applications using Julia.